scripts/tests/provisioning/test_provisioning.sh

```diff
@@ -9,7 +9,29 @@
 #  * evaluates the ptest results.
 
 script_dir="$(dirname "$0")"
-source "$script_dir/start-vm-using-expect.sh"
```

Comment:
Do you need to add `set -euo pipefail` here?

Reply:
That's a good idea. Will fix.